modules/mail_parser.py

```diff
@@ -10,7 +12,8 @@ def __init__(self):
         pass
 
     def get_urls(self, data):
-        return re.findall('http[s]?://(?:[a-zA-Z]|[0-9]|[$-_@.&+]|[!*\(\),]|(?:%[0-9a-fA-F][0-9a-fA-F]))+', data)
+        urls = re.findall('http[s]?://(?:[a-zA-Z]|[0-9]|[$-_@.&+]|[!*\(\),]|(?:%[0-9a-fA-F][0-9a-fA-F]))+', data)
```

Comment:
yes, I plan on using BeautifulSoup for parsing HTML so this will probably remain only for plaintext emails. let's not optimize anything atm